### ISSUE TITLE
Consider to move into the Flux organisation

### ISSUE THREAD

**Issue by XVilka** [yuehhua/GeometricFlux.jl#2]
For better visibility and future integration.

**Comment by yuehhua** [comment]
Sure! I am very willing to move this project under the organisation.
What can I do next?

**Comment by XVilka** [comment]
I think @ViralBShah @MikeInnes @oxinabox @tejank10 @avik-pal might help you do the [transfer](https://help.github.com/en/github/administering-a-repository/transferring-a-repository).

**Comment by ViralBShah** [comment]
@MikeInnes should be able to help with that. Feel free to ping me if still pending in a few days, and I can when I am not travelling.

**Comment by MikeInnes** [comment]
Is there some discussion of this that I missed?

I'd be open to this but I think the repo is currently a bit lacking on e.g. the docs and tutorials front.

**Comment by yuehhua** [comment]
Yes, I am currently preparing docs and tutorials, but there are some unstable about CuArrays and related dependencies.
So, I plan to wait for stable functional release, then I can release v0.2.0 and put on the docs as well as tutorials.
Meanwhile, I transfer this repo to FluxML. Any ideas?

**Comment by MikeInnes** [comment]
We're doing a round of tagging on the CuArrays and Zygote side, so lets get that done and take a look at the docs and tutorials etc. Then we can look at moving to FluxML, having a release announcement etc.

**Comment by yuehhua** [comment]
OK, I am working on #3.

**Comment by XVilka** [comment]
I believe GeometricFlux is ready to be considered stable enough to join FluxML organization. CuArrays.jl and Zygote.jl got a few releases since that was proposed. What do you think?

**Comment by ViralBShah** [comment]
Yeah good to revisit this.

**Comment by yuehhua** [comment]
It's almost stable. Just let me finish my docs...

**Comment by EricForgy** [comment]
Bump 😊

**Comment by yuehhua** [comment]
Done.

**Comment by XVilka** [comment]
Congratulations!

**Comment by ViralBShah** [comment]
Great to have the project in FluxML! I've invited you to be a member of FluxML and please let me or one of the org owners know if you need admin bit, etc. Always trying to catch up with whatever Github's new model is.

**Comment by VarLad** [comment]
Hello!
Has the merge been done?
